src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import './TodoList.scss';
+import { Todo } from './Todo';
+
+export class TodoList extends React.Component {
+  state = {
+    todos: [...this.props.todos],
+    title: '',
+    userName: '',
+    isTitleAdd: true,
+    isUserSelected: true,
+  };
+
+  handleChangeTitle = (event) => {
+    const { value } = event.target;
+
+    this.setState({
+      title: value,
+      isTitleAdd: true,
+    });
+  };
+
+  handleChangeUser = (event) => {
+    const { value } = event.target;
+
+    this.setState({
+      userName: value,
+      isUserSelected: true,
+    });
+  };
+
+  addTodo = (event) => {
+    event.preventDefault();
+
+    const { users } = this.props;
+
+    this.setState(({ todos, title, userName }) => {
+      if (!title && !userName) {
+        return {
+          isTitleAdd: false, isUserSelected: false,
+        };
+      }
+
+      if (!title || title.length > 35 || title.length < 5) {
+        return { isTitleAdd: false };
+      }
+
+      if (!userName) {
+        return { isUserSelected: false };
+      }
+
```

Comment:
Maybe move these logic to handlers

Reply:
Well, these checks should be triggered on form submit, so let them be here